suites/quincy/baremetal_pipeline/tier-1_rgw_ms.yaml

```diff
@@ -451,6 +451,41 @@ tests:
             name: push cosbench push workload from secondary
             polarion-id: CEPH-83575073
 
+  - test:
+      abort-on-fail: true
+      clusters:
+        ceph-pri:
+          config:
+            controllers:
+              - folio01
+            drivers:
+              - folio01
+              - folio02
+              - folio03
+            workload_type: fill_runtime
+            run_time: 7200
+            record_sync_on_site: ceph-sec
+            bucket_prefix: cosbench01-bkt-
+            number_of_buckets: 1
+            number_of_objects: 2000000
```

Comment:
@anrao19 , please add support for object starting number as well
that way we can split the workload between many clusters and trigger a cleanup workload for the particular range of objects

Reply:
@hmaheswa for clean up we are taking object count from bucket stats

Reply:
@anrao19 we can have the facility of splitting the cleanup workload as well
by specifying the objects starting number (and ending number or objects count)

Reply:
Changes made @hmaheswa